docs/UserGuide.md

```diff
@@ -80,22 +80,24 @@ or skip to the [command summary](#command-summary).
   + [Navigating the GUI](#navigating-the-gui)
   + [Tutorials](#tutorials)
 + [Features](#features)
-  + [Modifying Contact Data](#adding-a-contact-add)
+  + [Modifying Contact Data](#modifying-contact-data)
```

Comment:
Perhaps it would be better to name this section "Modifying Commands"?

Reply:
Yes, it has been changed.